.circleci/docker/Dockerfile.ubuntu1904

```diff
@@ -46,16 +46,6 @@ RUN set -ex; \
 
 FROM base AS libraries
 
-# Z3
```

Comment:
I didn't notice that this was in the docker image before - this build basically overwrote the static z3 lib installed above...